meilix-artwork_1.0-1_amd64.build

```diff
@@ -0,0 +1,112 @@
+ dpkg-buildpackage -rfakeroot -us -uc -ui
+dpkg-buildpackage: info: source package meilix-artwork
+dpkg-buildpackage: info: source version 1.0-1
+dpkg-buildpackage: info: source distribution unstable
+dpkg-buildpackage: info: source changed by Meilix <mb@mariobehling.de>
+ dpkg-source --before-build meilix-artwork
+dpkg-buildpackage: info: host architecture amd64
+ fakeroot debian/rules clean
+dh clean 
+dh: Compatibility levels before 9 are deprecated (level 8 in use)
+   dh_auto_clean
+dh_auto_clean: Compatibility levels before 9 are deprecated (level 8 in use)
+   dh_clean
+dh_clean: Compatibility levels before 9 are deprecated (level 8 in use)
+ dpkg-source -b meilix-artwork
+dpkg-source: warning: native package version may not have a revision
+dpkg-source: info: using source format '3.0 (native)'
+dpkg-source: info: building meilix-artwork in meilix-artwork_1.0-1.tar.xz
+dpkg-source: info: building meilix-artwork in meilix-artwork_1.0-1.dsc
+ debian/rules build
+dh build 
+dh: Compatibility levels before 9 are deprecated (level 8 in use)
+   dh_update_autotools_config
+   dh_auto_configure
+dh_auto_configure: Compatibility levels before 9 are deprecated (level 8 in use)
+   debian/rules override_dh_auto_build
+make[1]: Entering directory '/home/pranav1698/Desktop/open-source/meilix/meilix-artwork/meilix-artwork'
```

Comment:
This file contains lots of address of your personal system.

I think this file will get deleted and new one will generate, so why don't we use debuild script for the same.

Did you get my point? @pranav1698

Reply:
Yeah, I think this file gets generated every time we run debuild.sh, maybe deleting the generated files just for merging will be make the task clear

Reply:
Maybe add later to .gitignore and remove the file.